lamindb/_context.py

```diff
@@ -336,20 +351,24 @@ def _track_notebook(
             if _env in ("lab", "notebook"):
                 cls._notebook_meta = metadata  # type: ignore
             else:
-                msg = (
-                    "\n(1) Save your notebook!"
-                    "\n(2) Attach metadata to the notebook by running the CLI:\n"
-                    f"lamin track {notebook_path}"
-                    "\n(3) Reload or re-open your notebook"
-                )
+                msg = msg_manual_init.format(notebook_path=notebook_path)
                 raise UpdateNbWithNonInteractiveEditorError(msg)
 
         if _env in ("lab", "notebook"):
             # save the notebook in case that title was updated
             # but notebook not saved
             nbproject.dev._frontend_commands._save_notebook(_env)
+            # check here if interactivity really works
 
         if metadata is not None:
+            # this only executed if nbproject is used
+            # check here if interactivity really works
+            # timestamp should be recent due to save
+            is_interactive = _seconds_modified(_filepath) < 1.5  # should be ~1 sec
```

Comment:
I think this is a fine way of doing it, but wouldn't it be even more robust to compare the timestamp prior to saving with after saving and making sure it differs?

This here could fail in some cases. But it's also fine for me to keep it with a comment that we might want to make it more robust in the future.

Reply:
Hm, i am not really sure it is more reliable, because autosave can happen, for example.

Reply:
You're right that this should be fine! After all, we just want to make sure that the notebook just got saved.